builder/relinker/relinker.py

```
@@ -0,0 +1,358 @@
+        self.fsecs = func2sect(fsecs)
+        self.desc  = '*%s:%s.*'%(lib, file.split('.')[0])
+
+        secs = lib_secs(lib, file, lib_path)
+        if '.iram1.' in self.fsecs[0]:
+            self.secs = filter_secs(secs, ('.iram1.', ))
+        else:
+            self.secs = filter_secs(secs, ('.iram1.', '.text.', '.literal.'))
+        self.isecs = strip_secs(self.secs, self.fsecs)
+
+    def __str__(self):
+        s = 'lib=%s\nfile=%s\lib_path=%s\ndesc=%s\nsecs=%s\nfsecs=%s\nisecs=%s\n'%(\
```

Ruff (0.15.7)

[warning] Invalid escape sequence: `\l`

Add backslash to escape sequence

(W605)